scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -168,6 +168,8 @@ build-implementers-guide:
     - .docker-env
     - .test-refs
     - .collect-artifacts-short
+  variables:
+    GIT_DEPTH:                     0
```

Comment:
This should do the trick!

nit: It would be great to add a comment here. No one will remember why this is here 1 year from now.

Reply:
Agree, added the comment